### DISCUSSION TITLE
看了下源码，每个串口都开一个读线程，假如服务器的串口比较多，咋办？

### DISCUSSION THREAD

**Discussion by qbq600** [itas109/CSerialPort#66] [General]
冒昧问下大佬，看了下源码，每个串口都开一个读线程，假如服务器的串口比较多，咋办？

**Comment by itas109** [comment]
使用方面，多个串口多个类就可以读写。
性能方面，异步的实现是另开一个读线程，如果不需要异步，可以使用同步方式，这样不会额外增加一个线程。

也可以加群沟通。
QQ群 : [129518033](http://shang.qq.com/wpa/qunwpa?idkey=2888fa15c4513e6bfb9347052f36e437d919b2377161862948b2a49576679fc6)
